app/assets/javascripts/voting.js

```diff
@@ -0,0 +1,101 @@
+var makeAjaxRequestPost = function(buttonEle, reviewId, voteScore) {
+  var request = $.ajax({
+    method: 'POST',
+    data: { score: voteScore },
+    url: '/api/v1/reviews/' + reviewId + '/votes'
+  });
+
+  request.success(function(data) {
+    var buttons = $(buttonEle).closest('.vote_buttons_create');
+    $(buttons).removeClass('vote_buttons_create');
+    $(buttons).addClass('vote_buttons_update');
+    var voteButtons = $(buttons).children();
+    for (var i = 0; i < voteButtons.length; i++) {
+      $(voteButtons[i]).attr('action', '/reviews/' + reviewId + '/vote/' + data['vote']['id']);
```

Comment:
Line is too long.<br>['vote'] is better written in dot notation.<br>['id'] is better written in dot notation.